raiden-dapp/src/components/TokenOverlay.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <v-overlay :value="show" absolute opacity="1.0" class="token-network-overlay">
+    <v-container class="container">
+      <v-row no-gutters justify="end">
+        <v-btn @click="cancel" icon class="close-button">
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-row>
+
+      <v-row id="connectNew">
```

Comment:
Would it be possible to remove camelCase from the id? usually so far in templates, camelCase is either used for a variable or a method.